test.ini

```diff
@@ -4,7 +4,7 @@ host = 0.0.0.0
 port = 5000
 
 [app:main]
-use = config:./ckan/test-core.ini
+use = config:/usr/lib/ckan/default/src/ckan/test-core.ini
```

Comment:
There is a reason for this line! Please, do not change it or Jenkins will fail. You can modify it in your local system, but do not commit it!